content/2020-09-28-dicas-de-python-importando-modulos-no-python.md

```diff
@@ -0,0 +1,105 @@
+Title: Dicas de Python: Importando módulos no Python
+Slug: dicas-de-python-importando-modulos-no-python
+Date: 2020-09-28 21:28:08
+Tags: dicas, import, módulos
+Author: danielle8farias
+Comments: true
+
+
+
+## O que são módulos?
+
+De maneira resumida: um módulo é um trecho de código pronto que pode ser invocado quando for preciso utilizar algum método dele.
+
+Para ilustrar, vamos imaginar que exista no Python um **módulo de verduras e leguminosas** chamado de **vegetais** como na imagem abaixo:
+
+![vegetais]({filename}/images/vegetable.jpg)
+
+Se usarmos o comando:
+
+```
+import vegetais
+```
+
+O Python nos trará todos os vegetais mostrado. Como se colocássemos todos sobre a mesa para que durante o preparo, pudéssemos pegá-los de imediato.
+
+Porém ao usarmos o comando:
+
+```
+from vegetais import batata
+```
+
+O Python nos trará apenas a batata. Como se colocássemos sobre a mesa apenas a batata e mais nenhum outro vegetal. Sendo assim nos sobra mais espaço na mesa para outras funções.
+
+## Usando o import
+
+Usando um exemplo com Python, vamos importar o módulo **math**. Que é um módulo que nos dá acesso a várias funções matemáticas, como raiz quadrada, seno, arredondamentos, etc. 
+
+> você sempre pode consultar a [documentação do Python](https://docs.python.org/3/library/math.html) para saber mais sobre o módulo math e outros.
+
+O módulo **math** é um dos módulos conhecidos como *built in* (pronto para uso); basta apenas chamá-lo.
```

Comment:
Aqui, eu acho que eu daria exemplos de outros módulos que são built-in e de módulos que não são. Só pra exemplificar mesmo, tipo

```suggestion
O módulo **math** é um dos módulos conhecidos como *built in* (pronto para uso); ou seja, podemos usá-lo sem instalar nenhuma dependência externa, uma vez que fazem parte da biblioteca padrão do Python. Outros exemplos de módulos da biblioteca padrão são [os](https://docs.python.org/pt-br/3/library/os.html) (responsável por funções de sistema operacional) e [re](https://docs.python.org/pt-br/3/library/re.html) (expressões regulares).
```